package.json

```diff
@@ -69,6 +69,14 @@
       "es2015"
     ]
   },
+  "nyc": {
+    "extension": [
+      ".raw"
```

Comment:
I thought so too, at first, but the `*.js` files got included without problem.
But I'll do so, for the sake of completeness!